### ISSUE TITLE
Routine raku for evaluable representation of an object

### ISSUE THREAD

**Issue by uzluisf** [LaurentRosenfeld/think_raku#2]
In Chapter 1:
https://github.com/LaurentRosenfeld/think_raku/blob/b137b3b2247b5dba3ee582759c08f1e32f7c8ca1/book/chapt1.tex#L423

Since [Rakudo 2019.11](https://github.com/rakudo/rakudo/releases/tag/2019.11), you can use `.raku`, although it makes sense to use `.perl` since the Rakudo version listed in the book is 2019.03:

https://github.com/LaurentRosenfeld/think_raku/blob/b137b3b2247b5dba3ee582759c08f1e32f7c8ca1/book/chapt1.tex#L142-L144

**Comment by LaurentRosenfeld** [comment]
Well, the documentation still has ".perl" and not ".raku," and I wasn't able to download a version more recent than 2019.03, as the Rakudo Star download site appears not to have been updated since then. I would really prefer to change it to 2019.11  if I can, and replace .perl by .raku.

**Comment by uzluisf** [comment]
There was a release for [Rakudo 2019.11](https://github.com/rakudo/rakudo/releases/tag/2019.11) but that's only the compiler and not the Rakudo Star distribution which is always behind the compiler. [There seems to be a candidate release for Rakudo Star 2019.07](https://colabti.org/irclogger/irclogger_log/raku?date=2020-01-03#l600), thus it might take awhile for Rakudo Star 2019.11 to be released. IIRC, Rakudo Star is released quarterly.

**Comment by LaurentRosenfeld** [comment]
Yeah, I had seen it, but this book is to a large extent intended for beginners, not for Linux hackers. Remember that probably 90% of the people use Windows, including myself for my main PC because of $job reasons (the only VPN I am authorized to use runs on Windows, although, ironically, I use it to then connect to development and production platforms that run mostly on RHEL), and also because I'm using for a personal hobby Canon photographic software that is available only for Mac and Win. 

More generally, I prefer to tell my readers to use Rakudo Star, which is so simple to install, rather than to tell them to go through the hassle of compiling a release (at least under Windows). I think it is a shame that the latest available Rakudo * version is so old, it gives the impression that it is no longer or not well maintained.

Anyway, for the book, I think I'll keep it as it is for the time being. It is an on-line book, so it will be easy to update when the new R* version is released. 

Of course, feel free to suggest the 2019.11 release and to use the ".raku" method instead of ".perl" in your translation if you prefer.

**Comment by uzluisf** [comment]
Wholeheartedly agree. It's better to err in the side of caution and a more tested distribution. Furthermore, `.perl` occurs only in a few places in the book and won't be deprecated until 6.f, and removed in 6.g.
